browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2123,11 +2123,11 @@ export class CommentSection extends CanvasSectionObject {
 		if (availableSpace < this.sectionProperties.commentWidth && !this.isCollapsed)
 			horizontalScroll = (app.activeDocument.fileSize.cX + this.sectionProperties.commentWidth) * app.pixelsToTwips * app.dpiScale;
 
-		if (lastY > app.activeDocument.fileSize.pY) {
+		if (lastY > app.activeDocument.fileSize.pY && app.activeDocument.activeView.type !== 'ViewLayoutMultiPage') {
 			app.activeDocument.activeView.viewSize = new cool.SimplePoint(horizontalScroll, lastY * app.pixelsToTwips);
```

Comment:
Can't we have method like `activeview.setMaxSizeY(lastY)` which will have different implementation in every view layout subclass?
It looks like anti-pattern to check type of subclass and do conditional calculations outside of these interested classes. Especially that it modifies it's internal details :)